<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTransferFunctionGraphicalViewBase</class>
 <widget class="QWidget" name="QTransferFunctionGraphicalViewBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="udg::QColorTransferFunctionGraphicalView" name="m_colorView"/>
   </item>
   <item>
    <widget class="udg::QOpacityTransferFunctionGraphicalView" name="m_scalarOpacityView"/>
   </item>
   <item>
    <widget class="QGraphicsView" name="m_gradientOpacityGraphicsView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::QColorTransferFunctionGraphicalView</class>
   <extends>QGraphicsView</extends>
   <header>qcolortransferfunctiongraphicalview.h</header>
  </customwidget>
  <customwidget>
   <class>udg::QOpacityTransferFunctionGraphicalView</class>
   <extends>QGraphicsView</extends>
   <header>qopacitytransferfunctiongraphicalview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
